src/main/scala/translating/SemanticsLoader.scala

```diff
@@ -579,15 +592,15 @@ class SemanticsLoader(parserMap: immutable.Map[String, Array[Array[StmtContext]]
     }
     val index = parseInt(ctx.index)
 
-    resolveArrayExpr(name, index)
+    resolveArrayExpr(name, index.toInt)
   }
 
   private def visitIdent(ctx: IdentContext): String = {
     ctx.ID.getText.stripPrefix("\"").stripSuffix("\"")
   }
 
-  private def visitInteger(ctx: IntegerContext): Int = {
-    ctx.DEC.getText.toInt
+  private def visitInteger(ctx: IntegerContext): BigInt = {
+    BigInt(ctx.DEC.getText, 10)
   }
```

Comment:
What's the purpose of this change?

Reply:
The opcode passed through unsupported_op  as an int isn't representible in a signed Int32.